draft-ietf-rats-architecture.md

```diff
@@ -373,60 +378,78 @@ Attesting Environments are designed specifically with claims collection in mind.
 ~~~~
 {:twotypes-env #twotypes-env title="Two Types of Environments"}
 
-## Layered Attestation Environments {#layered-attestation}
+## Layered Attestation Procedures {#layered-attestation}
 
 By definition, the Attester role takes on the duty to create Evidence.
-The fact that an Attester role is composed of environments that
+The fact that an Attester role is composed of several types of environments that
 can be nested or staged adds complexity to the architectural layout of how an
-Attester can be composed and therefore has to conduct the Claims collection
-in order to create believable attestation Evidence.
-
-{{layered}} depicts an example of a device that includes (A) a BIOS stored
-in read-only memory in this example, (B) an updatable bootloader, and (C)
-an operating system kernel.
+Attester -- in itself -- is composed and therefore has to conduct the Claims collection
```

Comment:
I find the wording of "--in itself-- is composed" to be harder to read than the previous text

Reply:
This change has been reverted.